src/main/java/neresources/nei/NEIOreHandler.java

```diff
@@ -109,6 +114,33 @@ public void drawExtras(int recipe)
     }
 
     @Override
+    public List<String> handleTooltip(GuiRecipe gui, List<String> currenttip, int recipe)
+    {
+        if (GuiContainerManager.shouldShowTooltip(gui) && currenttip.size() == 0)
+        {
+            Point offset = gui.getRecipePosition(recipe);
+            Point pos = getMousePosition();
```

Comment:
I don't like static imports ^^ So could you make it GuiDraw.getMousePosition() ?